appveyor.yml

```diff
@@ -184,12 +184,7 @@ build_script:
 after_test:
 - if [%WITH_COVERAGE%]==[1] %APPVEYOR_BUILD_FOLDER%/opencppcoverage_mergereports.bat
 - if [%WITH_COVERAGE%]==[1] python %APPVEYOR_BUILD_FOLDER%/fixcoveragefilepaths.py -i coverage.xml -s %APPVEYOR_BUILD_FOLDER% -o coverage_pathsfixed.xml
-# use codecov bash script. Requires powershell and bash to be installed, which both are on the AppVeyor machine image
-- ps: |
-    $env:PATH = 'C:\msys64\usr\bin;' + $env:PATH
-    Invoke-WebRequest -Uri 'https://codecov.io/bash' -OutFile codecov.sh
-    bash codecov.sh -f "./coverage_pathsfixed.xml"
-#- if [%WITH_COVERAGE%]==[1] codecov -X gcov -f %APPVEYOR_BUILD_FOLDER%/build/coverage_pathsfixed.xml
+- if [%WITH_COVERAGE%]==[1] codecov -X gcov -f %APPVEYOR_BUILD_FOLDER%/build/coverage_pathsfixed.xml
```

Comment:
I thought we moved to using the codecov bash script instead of the codecov python script?

Reply:
I want to see if revert back to run coverage analysis locally will work and get rid of the inconsistency with run coverage analysis on AppVeyor because of the open issue. Also found that codecov has problem with gcov-8 (https://community.codecov.io/t/inconsistent-coverage-data-from-gcov-8/218)